pkg/nfd-topology-updater/nfd-topology-updater.go

```diff
@@ -337,6 +339,38 @@ func (w *nfdTopologyUpdater) updateNodeResourceTopology(zoneInfo v1alpha2.ZoneLi
 	return nil
 }
 
+func getCPUCoreAttribute(coreKind string) (error, v1alpha2.AttributeInfo) {
```

Comment:
nit: the common convention in go is to return error as the last value. It's so widely adopted that the other way around feels wonky

```suggestion
func getCPUCoreAttribute(coreKind string) (v1alpha2.AttributeInfo, error) {
```